Dashboard/app/js/templates/navSurveys/question-view.handlebars

```diff
@@ -33,18 +33,36 @@
      <a {{action "showAddAttributeDialog" target="this"}} class="addAttribute"><span>+</span>{{t _create_new_tag}} </a> </label>
       {{#if view.attribute}}<label class="labelcheckbox">{{view Ember.Checkbox checkedBinding="view.includeInMap"}}{{t _include_in_map}}</label>{{/if}}
   {{/if}}
-    <label class="selectinLabel"><span>{{t _question_type}}:</span> {{view Ember.Select
-      contentBinding="FLOW.questionTypeControl.content"
-      optionLabelPath="content.label"
-      optionValuePath="content.value"
-      selectionBinding="view.type" }} </label>
+    {{#if view.amDateType}}
+      <label class="selectinLabel"><span>{{t _question_type}}: {{tooltip _question_type_date_tooltip}}</span> {{view Ember.Select
+        contentBinding="FLOW.questionTypeControl.content"
```

Comment:
@gabemart given that the tooltip is the main difference here perhaps you could have only a single `{{Ember.Select ...}}` and the `{{#if ... }}` statements would only apply specifically to the tooltip part? that way we avoid duplicating the same code.